source/jsonnet/northern-ireland/individual/blocks/employment/main_employment_block.jsonnet

```diff
@@ -10,7 +10,10 @@ local rules = import 'rules.libsonnet';
         title: 'Main job',
         contents: [
           {
-            description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours.',
+            description: 'The next set of questions is about your last main job. This is the most recent job you had.',
```

Comment:
Might want to double check with @donnap573 but having checked the spreadsheet this change looks like it's supposed to be for the past variant?